src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -929,6 +929,14 @@ def load_arguments(self, _):
             help="Space-separated machine names to delete.",
         )
 
+    with self.argument_context("aks nodepool delete") as c:
+        c.argument(
+            "ignore_pod_disruption_budget",
+            options_list=["--ignore-pod-disruption-budget, --ignore-pdb"],
```

Comment:
You can add a rule exclusion to `src/azure-cli/azure/cli/command_modules/acs/linter_exclusions.yml`

Reply:
Added.